xds/xds.go

```diff
@@ -75,6 +78,7 @@ func init() {
 		v3statusgrpc.RegisterClientStatusDiscoveryServiceServer(grpcServer, csdss)
 		return csdss.Close, nil
 	})
+
```

Comment:
Nit: please delete.

Reply:
Ah, done. This was from trying to plumb resource not found through here.